app/coffee/Features/Email/EmailBuilder.coffee

```diff
@@ -111,16 +111,16 @@ Thank you
 """
 	compiledTemplate: (opts) -> 
 		SingleCTAEmailBody({
-			title: "#{ opts.project.name } &ndash; shared by #{ opts.owner.email }"
+			title: "#{ opts.project.name.slice(0, 40) } &ndash; shared by #{ opts.owner.email }"
```

Comment:
We should give some indication that the project name has been truncated. Although with the rate limiting apparently successful, I wonder if we can remove this again?